### PR TITLE
Help menu icons are not hyperlinked

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/15036

## Testing
**URL to test:** 
https://deploy-preview-1244--atd-moped-main.netlify.app/moped/

**Steps to test:**
1. Click on the help menu navbar, test clicking on the icons of each menu item. Clicking on the icon should redirect you. Also clicking on the very edge of the menu item should redirect you.
2. Now test on mobile. Before this update clicking on the very edge of the menu item wouldn't redirect you, now it should.

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/layouts/DashboardLayout/NavBar/DropdownMenu.js [hunk 1]
```diff
@@ -141,18 +141,18 @@ const DropdownMenu = ({
...
-          >
+        <Link
+          href={arcGISLink.link}
+          target="_blank"
+          rel="noopener noreferrer"
+          color="inherit"
+          underline="none"
+        >
+          <MenuItem onClick={handleDropdownClose}>
+            <ListItemIcon>{arcGISLink.Icon}</ListItemIcon>
```
Comment (roseeichelmann) [inline moped-editor/src/layouts/DashboardLayout/NavBar/DropdownMenu.js:152]: just moving the Link component outside of the MenuItem

File: moped-editor/src/layouts/DashboardLayout/NavBar/DropdownMenu.js [hunk 2]
```diff
@@ -162,20 +162,20 @@ const DropdownMenu = ({
         {helpItems.map((item) => {
           if (item.linkType === "external") {
             return (
-              <MenuItem key={item.link} onClick={handleDropdownClose}>
```
Comment (johnclary) [inline moped-editor/src/layouts/DashboardLayout/NavBar/DropdownMenu.js:165]: it looks like you need to move the `key` prop up to the `Link` component. i'm getting the warning:

```
DropdownMenu.js:116 Warning: Each child in a list should have a unique "key" prop.
```

**Review by johnclary — CHANGES_REQUESTED** [review]
Works beautifully! there's just a lil console error to squash 🙏

**Review by mddilley — APPROVED** [review]
So rad! Tested on the preview, and I'm redirected no matter where I click in the menu items. Thanks for making them more forgiving! 🙌 🚢

**Review by chiaberry — APPROVED** [review]
approval assuming you will squash that console error
and then we can click wherever we want on the menu!

**Review by johnclary — APPROVED** [review]
🚢 🚢 🚢 🚢 🚢 🚢 ✨

**Pull request merged**